.github/workflows/containers/github-action-ci/stage1.Dockerfile

```diff
@@ -19,13 +19,11 @@ RUN curl -O -L https://github.com/llvm/llvm-project/archive/refs/tags/llvmorg-$L
 
 WORKDIR /llvm-project-llvmorg-$LLVM_VERSION
```

Comment:
Can we rename this file to something other than `stage1` now that we only have a single stage?